galsim/utilities.py

```diff
@@ -418,3 +418,113 @@ def _gammafn(x):
                0.00000000000000000141, -0.00000000000000000023, 0.00000000000000000002
              )
 
+def interleaveImages(im_list,N,offsets,add_flux=True,suppress_warnings=False,catch_offset_errors=True):
+    """
+    Interleaves two or more images and outputs a larger image.
+
+    The sampling length of simulated images can be set arbitrarily using the `pixel_scale' argument
+    in drawImage() routine appropriately. However, pixel level detector effects can be included
+    only on images drawn at the native pixel scale, which are typically undersampled. Nyquist
+    sampled images that also include the effects of detector non-idealities can be obtained by
+    drawing multiple undersampled images (with the detector effects included) that are offset from
+    each other by a fraction of a pixel. This is equivalent to obtaining a finer sampled image from
+    a dither sequence, except that this routine handles only equispaced offsets. The dither sequence
+    must be a list of galsim.Image instances supplied through 'im_list'.
+
+    @param im_list             A list containing the galsim.Image instances to be interleaved.
+    @param N                   Number of images to interleave in either directions. It can be of
+                               type `int' if equal number of images are interleaved in both
+                               directions or a list or tuple of two integers, containing the number
+                               of images in x and y directions respectively.
+    @param offsets             A list containing the offsets as galsim.PositionD instances
```

Comment:
It might help to explicitly mention that the lower and upper limits for the offset values are +/- 0.5.  You could also give an example like, for N = 2, valid values for offsets must be some permutation of [galsim.PositionD(-0.25, -0.25), galsim.PositionD(-0.25, 0.25), galsim.PositionD(0.25, -0.25), galsim.PositionD(0.25, 0.25)].